.github/workflows/ci.yml

```diff
@@ -21,23 +21,54 @@ jobs:
       uses: actions/setup-python@v5
       with:
         python-version: ${{ matrix.python_version }}
+        cache: 'pip' # Cache pip dependencies
 
-    - name: Set up Pants
+    - name: Install uv
+      run: curl -LsSf https://astral.sh/uv/install.sh | sh
+
+    - name: Install build tool and dependencies
+      run: |
+        # Install the build tool
+        uv pip install --system build
+        # Install dev dependencies defined in the root pyproject.toml (if any)
+        # uv pip sync --dev # Uncomment if you have [tool.uv.dev-dependencies]
+
+    - name: Build all packages
       run: |
-        # Install Pants
-        export PATH=$PATH:/root/.local/bin
-        curl --proto '=https' --tlsv1.2 -fsSL https://static.pantsbuild.org/setup/get-pants.sh | bash
-        pants package ::
+        # Iterate through workspace members and build each one
+        # Extract members from root pyproject.toml (requires toml parser like yq or similar, or hardcode)
+        # For simplicity, listing them explicitly based on pyproject.toml:
+        PACKAGES=(
+            "src/edx_sysadmin"
```

Comment:
tomllib comes bundled with Python now.  Bet you could do this with a one liner and avoid the DRY violation.

Reply:
Here you go :)

```python
import tomllib; pypr=open("pyproject.toml","rb"); t=tomllib.load(pypr); members = t['tool']['uv']['workspace']['members']; print([ member.removeprefix('src/') for member in members ])
```